src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -7408,29 +7424,55 @@ def nunique(
         # Result is basically a series with the column labels as index and the distinct count as values
         # for each data column
         # frame holds rows with nunique values, but result must be a series so transpose single row
-        return self._nunique_columns(dropna).transpose_single_row()
+        return self._nunique_columns(
```

Comment:
since you have extended the other apis to include is_index parameter, why not stay consistent on all API extension on this?

Reply:
I'm no longer implementing nunique as a special case